cmake/compiler/clang/target_warnings.cmake

```diff
@@ -109,11 +109,25 @@ endmacro()
 #
 
 macro(toolchain_cc_warning_error_misra_sane dest_var_name)
-  set_ifndef(${dest_var_name} "-Werror=vla")
+  if (NOT ${dest_var_name})
+    set(${dest_var_name}
+      -Werror=vla
+      -Werror=implicit-fallthrough
+      -Werror=conversion
+      -Werror=override-init
+      )
+  endif()
 endmacro()
 
 macro(toolchain_cc_cpp_warning_error_misra_sane dest_var_name)
-  set_ifndef(${dest_var_name} "-Werror=vla")
+    if (NOT ${dest_var_name})
```

Comment:
indentation seems wrong.

Reply:
Have changed the way the flags were set an consequently didn't touch it. Would you mind review it again ?